src/components/HandleErrorsComponent.tsx

```diff
@@ -0,0 +1,49 @@
+import { FC } from 'react';
+import { AddingError } from './Notifications/AddingError';
+import { DeleteError } from './Notifications/DeleteError';
+import { ErrorLoadingComments } from './Notifications/ErrorLoadingComments';
+import { NoCommentsYet } from './Notifications/NoCommentsYet';
+
+type Props = {
+  isLoadingError: boolean,
+  isDeleteError: boolean,
+  isAddingError: boolean,
+  isNoComments: boolean,
+  isLoader: boolean,
+};
+
+export const HandleErrorsComponent: FC<Props> = ({
+  isLoadingError,
+  isDeleteError,
+  isAddingError,
+  isNoComments,
+  isLoader,
+}) => {
+  const isCommentsError = isNoComments && !isLoader;
+
+  const errorPairs = [
+    {
+      condition: isLoadingError,
+      errorComponent: <ErrorLoadingComments key={1} />,
```

Comment:
It looks strange that you use different components that look identical. You could just create one component for the error and change the title